pkg/apis/serving/v1alpha1/service_defaults_test.go

```diff
@@ -770,3 +773,108 @@ func TestServiceDefaulting(t *testing.T) {
 		})
 	}
 }
+
+func TestAnnotateUserInfo(t *testing.T) {
```

Comment:
Just for clarity: This is only moving the test from arguably the wrong file to here?

Reply:
Yes.